Traits/Reflection.php

```diff
@@ -81,4 +82,28 @@ public function getMethods() {
 	public function getProperties() {
 		return $this->_getProperties();
 	}
+
+	/**
+	 * (non-PHPdoc)
+	 * @see \Circle\UtilityBundle\Traits\Reflection::_getTraits()
+	 */
+	public function getTraits($deep = false) {
```

Comment:
das is die stelle, ne? Also erstmal en paar fragen: Warum brauchs du die unterscheidung, ob deep oder nich? Was spricht dagegen ne methode getTraits und getTraitsFlat zu haben, wenn du das wirklich brauchs? würde das Problem nich durch ne andere Datenstruktur gelöst werden (->baum zB auf dem ich dann abfragen machen kann, wie "gib mir nur ebene 1"<-das wäre flat)?